litestar/cli/commands/core.py

```diff
@@ -189,7 +189,7 @@ def run_command(
 
     if pdb:
         os.environ["LITESTAR_PDB"] = "1"
-
+    quiet_console = os.getenv("LITESTAR_QUIET_CONSOLE") or False
```

Comment:
Can we expose this from `_utils.py` instead of having another call to `getenv()` like we do the other below?